packages/fx-core/src/core/middleware/utils/appYmlGenerator.ts

```diff
@@ -112,10 +112,10 @@ export class AppYmlGenerator extends BaseAppYmlGenerator {
       this.handlebarsContext.aadAppName = aadManifest.name;
     }
 
-    const teamsAppManifestPath = path.join(this.projectPath, "appPackage/manifest.template.json");
+    const teamsAppManifestPath = path.join(this.projectPath, "appPackage/manifest.json");
```

Comment:
we may use the `Constants.teamsManifestFileName` constant exported from `projectMigratorV3.ts`

Reply:
Updated, thx!